src/main/java/io/appium/java_client/service/local/AppiumDriverLocalService.java

```diff
@@ -94,10 +94,23 @@ public static AppiumDriverLocalService buildService(AppiumServiceBuilder builder
     }
 
     public AppiumDriverLocalService withBasePath(String basePath) {
-        this.basePath = basePath;
+        this.basePath = sanitizeBasePath(basePath);
         return this;
     }
 
+    @SneakyThrows private static String sanitizeBasePath(String basePath) {
```

Comment:
what does this method throw?

Reply:
I will remove the SneakyThrows, it came by mistakenly copy paste. thanks for highlighting.